src/System.Security.Cryptography.Encoding/tests/AsnEncodedData.cs

```diff
@@ -107,7 +107,7 @@ public static void TestSubjectAlternativeName_Unix()
                 // Skip Choice[4]: DirName
                 //   (Supported by OpenSSL, but not by our Apple version)
                 // Choice[5]: EdiName
-                "EdiPartyName:<unsupported>",
+                // "EdiPartyName:<unsupported>",
```

Comment:
Rather than take the EdiPartyName out, can we add one in that parses so we don't lose code coverage?

Reply:
I'll try to find some real-world data for it. I prefer that to constructing something myself based on the untested definitions.

Reply:
`A5 0C 81 0A 13 08 65 64  69 50 61 72 74 79` (https://bugzilla.mozilla.org/show_bug.cgi?id=233586) should do it. I will update the PR later with that one.

Reply:
...and sure enough the definition is broken :-)

Reply:
The updated definition:
```c#
    [StructLayout(LayoutKind.Sequential)]
    internal struct EdiPartyNameAsn
    {
        [OptionalValue]
        [ExpectedTag(0, ExplicitTag = true)]
        internal DirectoryStringAsn? NameAssigner;

        [ExpectedTag(1, ExplicitTag = true)]
        internal DirectoryStringAsn PartyName;
    }
```

still fails on the above data. The test data uses implicit tags, which is probably wrong for DictionaryString (a choice). I will try to find some other test data.

Reply:
Ok, fixed, along with a test case. After reading the Mozilla bug linked above it became obvious to me that I actually copied the broken value before they fixed the bug.